gitpod-io/website
# Fix demo button on mobile navigation

## Description

Minor spacing fix for the demo button on mobile navigation.

| Dashboard | Demo (BEFORE) | Demo (AFTER) |
|-|-|-|
| <img width="496" alt="button-dashboard" src="https://user-images.githubusercontent.com/120486/222737531-e091acb9-b3a7-4c6c-b97c-edf80242a539.png"> | <img width="496" alt="button-demo-before" src="https://user-images.githubusercontent.com/120486/222737539-ebac388a-bd22-42a1-9f52-a1c01983d66b.png"> | <img width="496" alt="button-demo-after" src="https://user-images.githubusercontent.com/120486/222737544-1c87331e-f30f-4e37-aa7e-6d69b0c3ef95.png"> |

## How to test

Open the navigation on mobile while not being signed in the dashboard and notice how the demo button is using the same spacing as similar buttons.
